src/app/container/launch/tool-launch.service.spec.ts

```diff
@@ -57,4 +56,10 @@ describe('ToolLaunchService', () => {
         expect(service.getDockstoreSupportedCwlMakeTemplateString('quay.io/briandoconnor/dockstore-tool-bamstats', '1.25-11')).toContain(
             'cwltool --make-template quay.io/briandoconnor/dockstore-tool-bamstats:1.25-11 > input.yaml');
     }));
+    it('should get the right check tool command', inject([ToolLaunchService], (service: ToolLaunchService) => {
+        expect(service).toBeTruthy();
+        expect(service.getCheckToolString('potato', null)).toBe('$ dockstore tool check --entry potato checkparam.json');
+        expect(service.getCheckToolString('potato', 'stew')).toBe('$ dockstore tool check --entry potato:stew checkparam.json');
```

Comment:
I think these should be dockstore checker ... instead of dockstore tool check...

Reply:
I'm following the specs which say:
```
# runs the original tool or workflow
dockstore tool launch --entry sample_workflow:1.2 --json param.json
dockstore workflow launch --entry sample_tool:1.2 --json param.json
# runs the checker workflow
dockstore tool check --entry sample_workflow:1.2 --json param.json
dockstore workflow check --entry sample_tool:1.2 --json param.json
```

Reply:
I'm ok with either, I think we were leaning toward `dockstore checker` to group all the chcker related commands/code.  This will need to be consistent between the UI and the CLi though obviously